### PR TITLE
Added existingVnet and existingSubnet as parameter

### PR DESCRIPTION
**Pull request by fberson**
Added existingVnet and existingSubnet as parameter

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@fberson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@fberson__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: rds-deployment-ha-gateway/azuredeploy.parameters.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
       "value": "GEN-UNIQUE"
     },
     "storageAccountName": {
-      "value": "StorageAccount-Name"
+      "value": "storageaccount"
```
Comment (gatneil) [inline rds-deployment-ha-gateway/azuredeploy.parameters.json]: please use GEN-UNIQUE

**Comment by gatneil** [comment]
Small fix: please use GEN-UNIQUE for storage account name in the parameters file

**Comment by gatneil** [comment]
looks good to me; merging

**Pull request merged**